donate/recurring.html

```diff
@@ -0,0 +1,46 @@
+---
+layout: default
+title: Donate
+---
+
+<form action="https://www.paypal.com/cgi-bin/webscr" method="post">
+   <input type="hidden" name="cmd" value="_xclick-subscriptions">
+
+   <!-- TO DO need to set the email to your PayPal email or Secure Merchant ID -->
+   <input type="hidden" name="business" value="accshakes@gmail.com">
+   <input type="hidden" name="lc" value="US">
+   <input type="hidden" name="item_name" value="Donate to My Organization">
+   <input type="hidden" name="item_number" value="1234">
+   <input type="hidden" name="no_note" value="1">
+   <input type="hidden" name="no_shipping" value="2">
+   <input type="hidden" name="src" value="1">
+   <input type="hidden" name="p3" value="1">
+   <input type="hidden" name="currency_code" value="USD">
+   <input type="hidden" name="bn" value="PP-SubscriptionsBF:btn_subscribeCC_LG.gif:NonHosted">
+   <table>
+       <tr><td>When would you like this to Recur?</td></tr><tr><td><select name="t3">
+       <option value="D">Daily</option>
+       <option value="M">Monthly</option>
+       <option value="Y">Yearly</option>
+   </select> </td></tr>
+       <tr><td>How many times would you like this to recur? (including this payment)</td></tr><tr><td><select name="srt">
```

Comment:
This line is very messy. Something like:

```
<tr> 
    <td> ... </td>
</tr>
<tr>
    <td>
        <select name="srt">
            ...
        </select>
    </td>
</tr>
```

Would be much better to look back at later. One level of nesting per nested element keeps us from issues with tags not having closures/not knowing what's nested in what.

This is the same styling we'll enforce for the entire project.